homework/queries.sql

```diff
@@ -1,19 +1,100 @@
---- Get the top 3 product types that have proven most profitable
-
+\c ds_assignment2;
 
+--- Get the top 3 product types that have proven most profitable
+select d1.product_line as Most_profitable_lines
+	from products_info_dim as d1
+	join measures as m
+	on m.product_code = d1.product_code
+	group by d1.product_line
+	order by sum(m.profits) desc
+	limit(3);
+	
 --- Get the top 3 products by most items sold
-
+select d1.product_name as Most_sold_products
+	from products_info_dim as d1
+	join measures as m
+	on m.product_code = d1.product_code
+	group by d1.product_code
+	order by sum(m.quantity_ordered) desc
+	limit(3);
 
 --- Get the top 3 products by items sold per country of customer for: USA, Spain, Belgium
-
+(select d1.product_name as Most_sold_products_country, c.country
```

Comment:
It works, but you should feel uncomfortable with this... If we wanted 6 countries or 12 or all the countries...?